### PR TITLE
Adding extensions for SAW build

### PR DESCRIPTION
**Pull request by joberume**
This installs extensions needed for the SAW build. Artifact contains pre-built extensions to be packaged with the build.

### CODE REVIEW

File: build/azure-pipelines/win32/sql-product-build-win32.yml [hunk 1]
```diff
@@ -114,6 +114,19 @@ steps:
...
+  - powershell: |
+      # Install SAW specific extensions
+      . build/azure-pipelines/win32/exec.ps1
+      $ErrorActionPreference = "Stop"
+      $tempFilePath = (New-TemporaryFile).FullName
+      $zipFilePath = $tempFilePath + ".zip"
+      $extensionUri = "$(standalone-extensions-uri)"
+      $adsExtensionPath = "$(agent.builddirectory)\azuredatastudio-win32-x64\resources\app\extensions"
+
+      exec { Invoke-WebRequest -Uri $extensionUri -OutFile $tempFilePath;Move-Item $tempFilePath $zipFilePath;Expand-Archive $zipFilePath -DestinationPath $adsExtensionPath; }
```
Comment (corivera) [inline build/azure-pipelines/win32/sql-product-build-win32.yml:126]: Do we need the Move-Item part here? Could we use the zipFilePath for the OutFile argument of the web request?

Comment (joberume) [reply build/azure-pipelines/win32/sql-product-build-win32.yml:126]: New-TempFile can't override extension. Always is .tmp. 

Expand-Archive can't deal with extensions other than .zip

**Review by corivera — APPROVED** [review]

File: build/azure-pipelines/win32/sql-product-build-win32.yml [hunk 2]
```diff
@@ -37,7 +37,7 @@ steps:
     inputs:
       azureSubscription: 'ClientToolsInfra_670062 (88d5392f-a34f-4769-b405-f597fc533613)'
       KeyVaultName: ado-secrets
-      SecretsFilter: 'github-distro-mixin-password'
+      SecretsFilter: 'github-distro-mixin-password,standalone-extensions-uri'
```
Comment (Charles-Gagnon) [inline build/azure-pipelines/win32/sql-product-build-win32.yml:40]: What is this URI?

In general extension packaging stuff like this should probably be done in a gulp task - that's a lot easier to work with and keeps these scripts from getting cluttered with too much extra stuff.

Comment (Charles-Gagnon) [reply build/azure-pipelines/win32/sql-product-build-win32.yml:40]: We already have a task to package in external extensions so it seems like extending that as needed would help keep the logic all together : https://github.com/microsoft/azuredatastudio/blob/main/build/azure-pipelines/mixin.js#L36

Comment (Charles-Gagnon) [reply build/azure-pipelines/win32/sql-product-build-win32.yml:40]: (that pulls from https://github.com/microsoft/azuredatastudio/blob/main/product.json#L76 if it isn't clear)

Comment (joberume) [reply build/azure-pipelines/win32/sql-product-build-win32.yml:40]: Thanks @Charles-Gagnon for the pointer on the gulp task! Will move things over there. As far as the market place, we probably won't use the marketplace gallery as what we have is internal only, so we don't want that exposed.

Comment (Charles-Gagnon) [reply build/azure-pipelines/win32/sql-product-build-win32.yml:40]: If you're doing that please use a clearer name - maybe something like `external-extensions-uri`

What extensions are these anyways?

**Comment by Charles-Gagnon** [comment]
@kburtram FYI. Do you have anyone on your team you'd like tagged for future build-related PRs by the way?

File: build/azure-pipelines/win32/sql-product-build-win32.yml (hunk 2 above)
Comment (joberume) [reply build/azure-pipelines/win32/sql-product-build-win32.yml:40]: Livesite extension, Simple PR

**Pull request closed without merging**